package.json

```diff
@@ -40,18 +42,20 @@
     "mongoose": "^5.0.2",
     "morgan": "^1.5.1",
     "natural": "^0.5.4",
-    "ogr2ogr": "jdesboeufs/ogr2ogr",
+    "ogr2ogr": "^1.3.0",
     "on-finished": "^2.3.0",
     "passport": "^0.4.0",
     "passport-oauth2": "^1.4.0",
     "plunger": "^0.1.3",
+    "pump": "^3.0.0",
     "request": "^2.79.0",
     "require-all": "^2.2.0",
     "rimraf": "^2.4.2",
     "superagent": "^3.5.2",
     "tldjs": "^2.0.0",
     "tmp": "0.0.31",
     "underscore.string": "^3.0",
+    "unzipper": "^0.9.3",
```

Comment:
Why not `decompress` ou `yauzl`? (It's just a question)

Reply:
![](https://badgen.net/packagephobia/publish/decompress?label=decompress)
![](https://badgen.net/packagephobia/publish/yauzl?label=yauzl)
![](https://badgen.net/packagephobia/publish/unzipper?label=unzipper)

Didn’t know `decompress`. Will switch. :)